_posts/2020-03-31-a-product-survey-story.markdown

```diff
@@ -0,0 +1,87 @@
+
+
+![What package manager do you use?](images/What-Package-manager-does-your-team-use.jpg)
+
+
+
+## How does MyGet fit into your DevOps lifecyle?  
+
+Over 98% of you responded that you are currently using source control for the proprietary code you write, which is a good thing: source control is a must-have for modern software development. Almost 100% of you who use source control are using Git, with Github, Azure DevOps (split between cloud and server), Bitbucket, GitLab, and Assembla coming in as your top five hosting platforms. \
+
+
+![What tools do you manage your source code with?](images/Whats-source-code-do-you-use.jpg)
+
+
+One of the most effective ways to leverage the artifacts you store in MyGet is to integrate MyGet with your CI/CD processes. More than 85% of you responded that you have already embraced the continuous delivery and integration, using a myriad of tools such as Azure DevOps, Jenkins, TeamCity, GitLab, Bamboo, Appveyor, and Travis CI to run your builds and push build artifacts to MyGet.
+
+
+![alt_text](images/What-do-you-use-CI-CD.jpg)
+
+
+
+## Why do you use MyGet?
+
+Because of its nature as a hub between multiple spokes of the SDLC, the number of different use cases for MyGet can be quite varied. That said, over 68% you reported using MyGet to share your private packages within your internal team. The next three most-reported use cases each had about the same number of responses:  23% of you use MyGet to store build artifacts, 21% responded that you use MyGet to control the open source packages used in your dev environment, followed closely by the 20% of you using MyGet to host open-source packages you have built and shared with the community/public. 
+
+
+![alt_text](images/What-is-your-primary-use.jpg)
+
+
+## What did you ask us to prioritize next?
+
+ \
+MyGet’s success depends on our ability to continually improve features and build new ones that make your lives easier and help you ship better products. Despite the rise and adoption of 3rd-party CI/CD platforms across the MyGet Community, more than 75% of you said that updating MyGet’s build services to support .NET Core 3 and C# 8 was still important to you. Improving uptime and availability, your second-most requested improvement, has been high on our priority list for a while as well, so we are glad to see that this aligns with the concerns you have. Your feedback has been enormously helpful as we’ve been culling down our roadmap for this next fiscal year. Thank you again for your candid input—we are going to be able to be much more focused on delivering the improvements that matter to you because of it!
```

Comment:
grammar edit I just caught:
`MyGet's success depends on our ability to continually improve and build new features that make your...`